<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerGui</class>
 <widget class="QMainWindow" name="ServerGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>549</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>715</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Server Gui - iBlitzkriegi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Console</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QSplitter" name="splitter">
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::DefaultContextMenu</enum>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="handleWidth">
               <number>1</number>
              </property>
              <property name="childrenCollapsible">
               <bool>true</bool>
              </property>
              <widget class="QListWidget" name="listWidget">
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
              <widget class="QScrollArea" name="scrollArea">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>69</width>
                  <height>188</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QLabel" name="output_label">
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>10</pointsize>
                     <weight>50</weight>
                     <bold>false</bold>
                     <strikeout>false</strikeout>
                     <kerning>true</kerning>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="checkbox_label">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Say</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="say_checkbox">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="input_lineedit">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="execute_button">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Execute</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="start_button">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="restart_button">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Restart</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="reload_button">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Reload</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="kill_button">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Kill</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="text">
                   <string>Ram Usage</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QProgressBar" name="ram_progress_bar">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="value">
                   <number>24</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="text">
                   <string>CPU Usage</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QProgressBar" name="cpu_progress_bar">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="value">
                   <number>24</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="text">
                   <string>GUI Usage</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QProgressBar" name="gui_usage_progress_bar">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="value">
                   <number>24</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Players</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Location</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time Joined</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Whitelisted</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>OP</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Startup</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="current_config_label">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Current Configuration: Vixio Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_server_config_button">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select different server configuration</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Java Version</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="java_version_combo_box"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="min_ram_label">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Min. Ram: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="max_ram_label">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Max. Ram: </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QSlider" name="min_ram_slider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="max_ram_slider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>JAR file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="jar_file_line_edit"/>
          </item>
          <item>
           <widget class="QPushButton" name="jar_file_button">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Custom arguments:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="custom_arguments_line_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="create_server_config_button">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Create new server configuration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="launch_server_button">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Launch Server</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
